.github/workflows/test-action.yaml

```diff
@@ -58,16 +58,17 @@ jobs:
         with:
           api-token: ${{ secrets.HEAL_API_TOKEN }}
           suite: "testcitriggers/heal-core-functions"
+          test-config: |
+            {
+              "entrypoint": "https://app-staging.heal.dev/",
+              "variables": {
+                "blockName": "Test block22"
+              }
+            }
           stories: |
             [
               {
-                "slug": "create-a-story-with-left-click",
-                "test-config": {
-                  "entrypoint": "https://app-staging.heal.dev",
-                  "variables": {
-                    "blockName": "Test block"
-                  }
-                }
+                "slug": "create-a-block-then-cleanup"
```

Comment:
why do we provide just this slug, here, instead of `create-a-story-with-left-click`? is it intended?

Reply:
i want to test if the entrypoint is set in test-config of suite level.